### ISSUE TITLE
Request for hyperparameter settings and extended experimental configurations

### ISSUE THREAD

**Issue by YanbeiYbe** [RCR-LYY/TimeCAP#2]
Dear TimeCAP authors,

Thank you for releasing the code for **TimeCAP: A Channel-Aware Pre-Training Framework for Multivariate Time Series Forecasting**.

I am currently trying to reproduce the reported results in the paper. In the experimental setup, the paper mentions that the configuration ranges are provided in the extended version. However, I could not find the extended version, appendix, or supplementary material in this repository.

Would it be possible to share the extended version, appendix, supplementary material, or the detailed hyperparameter settings used in the experiments?

In particular, I would like to ask about the following configurations:

1. **Model architecture settings**
   - model dimension
   - number of encoder layers
   - number of attention heads
   - feed-forward dimension
   - dropout rate

2. **Patch and channel-aware settings**
   - patch length
   - stride
   - channel grouping size
   - number of groups
   - meta-router related settings

3. **Training settings**
   - learning rate
   - batch size
   - number of training epochs
   - patience / early stopping
   - optimizer and weight decay
   - learning rate scheduler

4. **Mixed-supervision fine-tuning settings**
   - loss weights such as λ1, λ2, and λ3
   - settings for autoregressive head and one-shot decoding head

5. **Dynamic fusion settings**
   - sigmoid fusion parameters, such as α and β

6. **Experimental settings**
   - default configurations for zero-shot experiments
   - default configurations for few-shot experiments
   - default configurations for full-shot experiments
   - whether different datasets or prediction lengths use different hyperparameter settings

For reference, I am trying to reproduce the following setting:

```bash
python run.py --task_name finetune --downstream_task forecasting --model TimeCAP --data ETTh1 --seq_len 96 --pred_len 96
